server/graphql/v2/query/TransactionsQuery.ts

```diff
@@ -152,6 +152,16 @@ const TransactionsQuery = {
     if (host) {
       where.push({ HostCollectiveId: host.id });
     }
+
+    // No await needed, GraphQL will take care of it
+    // TODO: try to skip if it's not a requested field
+    const existingKinds = models.Transaction.findAll({
+      attributes: ['kind'],
+      where,
```

Comment:
Add where `kind IS NOT NULL`.
![Screenshot from 2021-09-06 11-11-04](https://user-images.githubusercontent.com/2119706/132231016-20f0840a-8fc3-4dfd-9009-150ad4547bd1.png)

Reply:
Or a proper label in the frontend (eg. `Unknown`) if we still want the ability to filter them